docs/plugins/export.rst

```diff
@@ -36,19 +39,34 @@ The ``export`` command has these command-line options:
 
 * ``--append``: Appends the data to the file instead of writing.
 
+* ``--format`` or ``-f``: Specifies the format the data will be exported as. If not informed, JSON will be used by default. The format options include csv, json and xml.
+
 Configuration
 -------------
 
 To configure the plugin, make a ``export:`` section in your configuration
-file. Under the ``json`` key, these options are available:
+file. Under the ``json``, ``csv``, and ``xml`` keys, these options are available:
+
+- **JSON Formatting**
+    - **ensure_ascii**: Escape non-ASCII characters with ``\uXXXX`` entities.
+
+    - **indent**: The number of spaces for indentation.
+
+    - **separators**: A ``[item_separator, dict_separator]`` tuple.
 
-- **ensure_ascii**: Escape non-ASCII characters with ``\uXXXX`` entities.
+    - **sort_keys**: Sorts the keys in JSON dictionaries.
 
-- **indent**: The number of spaces for indentation.
+- **CSV Formatting**
+    - **delimiter**: Used as the separating character between fields. The default value is a comma (,).
 
-- **separators**: A ``[item_separator, dict_separator]`` tuple.
+    - **dialect**: A dialect, in the context of reading and writing CSVs, is a construct that allows you to create, store, and re-use various formatting parameters for your data.
 
-- **sort_keys**: Sorts the keys in JSON dictionaries.
+- **XML Formatting**
+    - **encoding**: Use encoding="unicode" to generate a Unicode string (otherwise, a bytestring is generated).
```

Comment:
Hmm… how does this affect the usage of the plugin? Because we're writing to a file, it seems like the kind of Python string that gets generated is irrelevant. Perhaps there's an argument to be made that this should _not_ be configurable in that case?

Reply:
I agree, it is not an config option now.